code/datums/shuttles.dm

```diff
@@ -166,18 +166,18 @@
 	admin_notes = "No brig and no medical facilities. Build YOUR own."
 	credit_cost = 5000
 
-/datum/map_template/shuttle/emergency/construction_small
+/datum/map_template/shuttle/emergency/construction/small
 	suffix = "construction_small"
 	name = "Build Your Own Shuttle, Jr."
 	description = "The full-size BYOS too big for your taste? Aside from the reduced size and cost, this has the all same (lack of) amenities as its full-sized sibling."
 	admin_notes = "No brig and no medical facilities. Build YOUR own."
 	credit_cost = 2000
 
-/datum/map_template/shuttle/emergency/airless/prerequisites_met()
+/datum/map_template/shuttle/emergency/construction/prerequisites_met()
```

Comment:
Erm, doesn't that make the actual BYOS only buyable on first 10 minutes?

Reply:
small one's typepath was changed so it inherits this stuff